final/problem-def-ana/require-ana.tex

```diff
@@ -162,6 +171,8 @@ \subsubsection{Won't have}
 \subsection{Minimal viable product}
```

Comment:
This section feels a bit out of place right now.
Maybe that's just because it doesn't really add any new content.